init.lua

```diff
@@ -1,77 +1,302 @@
+local mod_name = "wielded_light"
+
 local update_interval = 0.2
+local cleanup_interval = update_interval*3
+local velocity_projection = update_interval * 1
 local level_delta = 2
```

Comment:
What does this do? Please add a short, helpful comment to the code.

Reply:
Which bit? `level_delta`? I didn't even add that...

Reply:
I think `velocity_projection` and `level_delta` would both benefit from a short comment. I do know that `level_delta` was not changed, but if you happen to know what it does, it would be a great opportunity to document it a bit.

Reply:
I've added in descriptions for all the constants at the top here